protocol/0080-SPOT-product_builtin_spot.md

```diff
@@ -0,0 +1,167 @@
+  - An LPs `virtual_stake` should be treated separately for each side of the book - call these the `buy_virtual_stake` and `sell_virtual_stake`.
+  - The same growth factor - as specified in the [LIQF spec](0042-LIQF-setting_fees_and_rewarding_lps.md) - derived from the `total value for fee purposes` in the quote asset is used to update both buy/sell virtual stakes (still in their respective assets).
+  - The current `virtual_stake` for fee splitting is the smaller of the two values where the `sell_virtual_stake` is converted to `quote_asset` at the current `mark_price`.
+
+From the above conditions, an LP is incentivised to provide a roughly equal value of liquidity on each side of the book at comparable levels of competitiveness in order to maximise their share of the liquidity fees.
+
+### Amendments and Cancellations
+
+A liquidity amendment or a cancellation is determined as valid following spec [0044-LIME](./0044-LIME-lp_mechanics.md) with the following exceptions:
+
+- the `maximum_reduction_amount` should be expressed in the `quote_asset` when reducing the `buy_commitment_amount` and the `base_asset` when reducing the `sell_commitment_amount`.
+- the `maximum_reduction_amount` should be `INF` in the case where the liquidity `time_window=Os` (Note: this is not strictly necessary but an LP is effectively able to reduce their commitment to zero anyway in this case through multiple commitments.)
+
+```pseudo
+Market Data:
+
+    base_asset: ETH
+    quote_asset: DAI
+
+    mark_price = 1000
+
+Market Liquidity:
+
+    total_stake = 100,000 DAI (or 100 ETH)
+    target_stake = 25,000 DAI (or 25 ETH)
+
+    maximum_reduction_amount = 1000,000 - 25,0000 = 75,000 DAI (or 75 ETH)
+```
+
+`virtual_stake` values should be updated during liquidity amendments or cancellations following the mechanisms detailed in spec [0044-LIME](./0044-LIME-lp_mechanics.md) with the following exceptions:
+
+- reducing the `buy_commitment_amount` only reduces the `buy_virtual_stake`
+- reducing the `sell_commitment_amount` only reduces the `sell_virtual_stake`
+
+A single LP will never be able to reduce their commitment to `0`. They can either keep reducing to a sufficiently small amount they're willing to ignore, or they can submit a governance vote to cancel the market, see the [governance spec](./0028-GOVE-governance.md).
+
+For market cancellation proposal a sole LP in the market holds all the voting power (unless governance token holders override them).
+
+### Liquidity Shortfalls
+
+If at any point in time, a liquidity provider has insufficient capital in their general accounts to cover a transfer arising from a filled liquidity order, the network will utilise the liquidity commitment, held in the relevant bond account to cover the shortfall, applying the bond penalty factor (slashing the bond).
+
+As there is no market insurance pool, funds from bond slashing in the result of shortfall will be transferred to the global insurance pool for that asset.
```

Comment:
there is also no such thing as global insurance pool for the asset. Maybe change to network treasury?

Reply:
Thanks, have updated to network treasury